### PR TITLE
Handle invalid zip files

### PR DESCRIPTION
**Pull request by rkondratowicz**
Move file to rejected container.
Previously file was being left where it was causing the job to process it on next run (and creating error event. over and over again).

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipVerifiers.java [hunk 1]
```diff
@@ -107,7 +108,7 @@ static ZipInputStream sha256WithRsaVerification(ZipStreamWithSignature zipWithSi
 
             return zipEntries;
         } catch (IOException ioe) {
-            throw new SignatureValidationException(ioe);
+            throw new InvalidZipArchiveException("Error extracting zip entries", ioe);
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipVerifiers.java]: nothing to do with signatures, hence new exception.
It extends `InvalidEnvelopeException` which is handled and results in moving file to rejected container.

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipVerifiers.java [hunk 2]
```diff
@@ -107,7 +108,7 @@ static ZipInputStream sha256WithRsaVerification(ZipStreamWithSignature zipWithSi
 
             return zipEntries;
         } catch (IOException ioe) {
-            throw new SignatureValidationException(ioe);
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipVerifiers.java]: Has `SignatureValidationException` or its construct became retired?

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipVerifiers.java]: still used in 5 other places in code.

**Comment by rkondratowicz** [comment]
closing, will create a new PR to start with a new AKS pipeline

**Pull request closed without merging**